src/handlers/sqs-indexer/handle-transfer-event.ts

```diff
@@ -51,7 +52,7 @@ async function handleTransferEvent(
       from,
       to,
       contractAddress,
-      tokenId: '0x' + remove0x(tokenId.toHexString()).padStart(64, '0'),
+      tokenId: hexZeroPad(tokenId.toHexString(), 32),
```

Comment:
@BravoNatalie what we have here originally is correct i believe. We ran into this as part of the weekend before dan witz. Whatever @ACC01ADE had before is correct. 

The main problem was that there ended up a missing 6 or 8 leading zeros.

1. `0x300000000...001`
2. `0x0000000300000000...001`